index.html

```diff
@@ -0,0 +1,54 @@
+<html>
+  <head>
+    <script src="jquery-2.1.4.min.js"></script>
+    <link rel="stylesheet" type="text/CSS" href="style.css">
+    <title>Rick's Blog</title>
+  </head>
```

Comment:
You're going to want a <meta name="viewport" ...> in the head section to make your site responsive.